webapp/channels/src/components/size_aware_image.tsx

```diff
@@ -158,7 +211,14 @@ export default class SizeAwareImage extends React.PureComponent {
     renderImageLoaderIfNeeded = () => {
         if (!this.state.loaded && this.props.showLoader && !this.state.error) {
             return (
-                <div style={{position: 'absolute', top: '50%', transform: 'translate(-50%, -50%)', left: '50%'}}>
+                <div
+                    style={{
+                        position: 'absolute',
+                        top: '50%',
+                        transform: 'translate(-50%, -50%)',
+                        left: '50%',
+                    }}
+                >
```

Comment:
can you please revert the additional formatting introduced? if linter didn't complain no need to change :)